### ISSUE TITLE
Patch error for fsync_futex_waitv

### ISSUE THREAD

**Issue by 4Evergreen4** [Frogging-Family/wine-tkg-git#1095]
Building wine-tkg-git on EndeavourOS (Arch Linux) with the default settings (0 - default-tkg) causes a patch error:

```
fsync_futex_waitv.patch -- ( Applied patches for fsync to support futex_waitv )
patching file configure
Hunk #1 succeeded at 7964 (offset -18 lines).
patching file configure.ac
Hunk #1 succeeded at 414 (offset -27 lines).
patching file dlls/ntdll/unix/esync.c
Hunk #1 succeeded at 952 (offset -1 lines).
Hunk #2 succeeded at 969 (offset -1 lines).
Hunk #3 succeeded at 1003 (offset -1 lines).
patching file dlls/ntdll/unix/file.c
Hunk #1 succeeded at 6492 (offset 1129 lines).
Hunk #2 succeeded at 6757 (offset 1129 lines).
Hunk #3 succeeded at 7589 (offset 535 lines).
Hunk #4 succeeded at 7617 (offset 535 lines).
patching file dlls/ntdll/unix/fsync.c
Hunk #3 succeeded at 57 (offset -1 lines).
Hunk #4 succeeded at 141 (offset -1 lines).
Hunk #5 succeeded at 693 (offset -1 lines).
Hunk #6 succeeded at 732 (offset -1 lines).
Hunk #7 succeeded at 846 (offset -1 lines).
Hunk #8 succeeded at 866 (offset -1 lines).
Hunk #9 succeeded at 892 (offset -1 lines).
Hunk #10 succeeded at 911 (offset -1 lines).
Hunk #11 succeeded at 932 (offset -1 lines).
Hunk #12 succeeded at 955 (offset -1 lines).
Hunk #13 succeeded at 967 (offset -1 lines).
Hunk #14 succeeded at 1072 (offset -1 lines).
Hunk #15 succeeded at 1092 (offset -1 lines).
Hunk #16 succeeded at 1137 (offset -1 lines).
patching file dlls/ntdll/unix/loader.c
Hunk #1 succeeded at 1010 with fuzz 2 (offset -1018 lines).
Hunk #2 succeeded at 1841 (offset -228 lines).
patching file dlls/ntdll/unix/unix_private.h
Hunk #1 FAILED at 146.
1 out of 1 hunk FAILED -- saving rejects to file dlls/ntdll/unix/unix_private.h.rej
patching file include/config.h.in
Hunk #1 succeeded at 150 (offset -38 lines).
```

Here are the full log files associated with the build:
[last_build_config.log](https://github.com/Frogging-Family/wine-tkg-git/files/13399095/last_build_config.log)
[prepare.log](https://github.com/Frogging-Family/wine-tkg-git/files/13399096/prepare.log)

**Comment by 4Evergreen4** [comment]
After doing a little poking it seems like the contents of `fsync_futex_waitv.patch` is not up to date with the latest wine-staging `unix_private.h` file (and probably others too that are hidden by that patch failing).

**Comment by rchris494** [comment]
yeah i am having the same problem with that patch

**Comment by gardotd426** [comment]
The problem is you can't even disable it. Customization.cfg has been changed and now even trying to build a clean git clone with default settings using the default tkg preset fails with that patch application.

**Comment by diomhaireachd** [comment]
The LAA patch toggle is in wine-tkg-profiles/advanced-customization.cfg _large_address_aware other patches that don't apply are fsync and fast sync.

**Comment by gardotd426** [comment]
Not sure what the LAA patch has to do with this.

**Comment by Tk-Glitch** [comment]
Fixed with https://github.com/Frogging-Family/wine-tkg-git/commit/9319a49a3b220a3626a8cb581fd8a99a990d84d8

**Comment by compicat** [comment]
> Fixed with [9319a49](https://github.com/Frogging-Family/wine-tkg-git/commit/9319a49a3b220a3626a8cb581fd8a99a990d84d8)

@Tk-Glitch Thank you. Can you please update winesync patch too? now it gives error during patching. Thanks in advance.
